client/common/services/canvas.service.js

```diff
@@ -2,14 +2,29 @@
     "use strict";
     
     angular
-        .module("services", [])
-        .service("CanvasService", CanvasService);
+        .module("smitty.services", [])
+        .service("canvasService", canvasService);
     
-    function CanvasService(){
-        
+    function canvasService() {
```

Comment:
This is the new `canvasService`. All attributes and operations for the `Canvas` should go here. Members prefixed with `this.` get exposed to your controllers.